test/integration/entrypoint/preserveOrder-entrypoint.test.js

```diff
@@ -0,0 +1,24 @@
+const {expect} = require('chai');
+
+
+describe('Entrypoint attribute preserveOrder', function () {
+    var testrun;
+
+    before(function (done) {
+        this.run({
+            collection: 'test/fixtures/Sample.postman_collection.json',
```

Comment:
Fixtures are hard to maintain and you have to open 2 files to understand the test.
So, we avoiding that for all the new tests.

I feel we don't need an integration test for this, unit tests should cover all the changes.

Reply:
Ok if thats the case I'll remove it.